README.md

```diff
@@ -72,7 +72,22 @@ So in this case, you'll want to add a login button to the `layout.html` of your
 
 `/auth/gitlab/login`
 
-That's all there is to it. When a user reaches this URL they are redirected to begin the authorization process with Gitlab, or the service of your choice.
+To generate these links automatically, use the `apos.passport.links` helper (assuming that you have set an `alias` of `passport` for this module):
```

Comment:
The alias hand wave is okay if we link to the documentation about it: https://v2.docs.apostrophecms.org/reference/core-server.html#special-property-names-in-module-configuration-objects

Reply:
The helper looks like it is incorrect here. Should be `loginLinks`